### PR TITLE
Adding balance details to WhatYouOwe page changes

### PR DESCRIPTION
**Pull request by chandraAndem**

### CODE REVIEW

File: public/stylesheets/itvc.css [hunk 1]
```diff
@@ -1665,3 +1665,23 @@ margin-bottom: 0 !important;
...
+
+.what-you-owe-total-values {
+    font-size: 36px !important;
+    line-height: 1.11 !important;
+    font-weight: bold;
+}
+
+.what-you-owe-total-values-bold {
+    font-weight: 700 !important;
+}
```
Comment (MihailsC-hmrc) [inline public/stylesheets/itvc.css:1687]: Just a note: these styles probably won't be required after migration to the new frontend system. Or probably they'll be replicated in new application.css during migration of the page.

File: app/models/financialDetails/FinancialDetailsResponseModel.scala [hunk 2]
```diff
@@ -91,7 +93,7 @@ case class FinancialDetailsModel(documentDetails: List[DocumentDetail],
   }
 
   def merge(financialDetailsModel: FinancialDetailsModel): FinancialDetailsModel = {
-    FinancialDetailsModel(documentDetails ++ financialDetailsModel.documentDetails,
+    FinancialDetailsModel(balanceDetails, documentDetails ++ financialDetailsModel.documentDetails,
```
Comment (MihailsC-hmrc) [inline app/models/financialDetails/FinancialDetailsResponseModel.scala:96]: Debatable: this function does not merge `balanceDetails` anyhow. Would it be more feasible to rename it to something like `def mergeLists`?

**Review by MihailsC-hmrc — COMMENTED** [review]
Need to clarify a question before approving the PR.

File: public/stylesheets/itvc.css (hunk 1 above)
Comment (chandraAndem) [reply public/stylesheets/itvc.css:1687]: Yes agree, will let Jack who is working on migration know about it.

File: app/models/financialDetails/FinancialDetailsResponseModel.scala (hunk 2 above)
Comment (chandraAndem) [reply app/models/financialDetails/FinancialDetailsResponseModel.scala:96]: Agreed and made those changes.

**Review by MihailsC-hmrc — APPROVED** [review]

**Pull request merged**